tests/cupyx_tests/scipy_tests/stats_tests/test_morestats.py

```diff
@@ -141,3 +141,8 @@ def test_array_empty_neg_lmb(self, xp, scp, dtype):
             return numpy.array(result)
         else:
             return result
+
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=atol, rtol=rtol)
```

Comment:
Could you add this condition?

```suggestion
    @testing.with_requires('scipy>=1.13')
    @testing.numpy_cupy_allclose(scipy_name='scp', atol=atol, rtol=rtol)
```